setup.cfg

```
@@ -38,8 +38,8 @@ install_requires =
     typing-extensions>=3.10.0.0
     packaging>=20.9
     htmltools>=0.2.0
-    # shiny>0.7.1
-    shiny @ git+https://github.com/posit-dev/py-shiny.git#egg=shiny
+    # shiny>=0.8.1
+    shiny @ git+https://github.com/posit-dev/py-shiny.git
```

```suggestion
    shiny>=0.8.1
```

* After posit-dev/py-shiny#1199